packages/tables/src/Concerns/CanPaginateRecords.php

```diff
@@ -69,6 +78,11 @@ protected function isTablePaginationEnabled(): bool
         return true;
     }
 
+    protected function isTablePaginationPerPageAllOptionEnabled(): bool
```

Comment:
What's the reason behind having this method instead of just returning the option in the array from `getTableRecordsPerPageSelectOptions`?